### PR TITLE
Add vertical geoid grids for Denmark, Faroe Islands and Greenland.

### PR DESCRIPTION
**Pull request by kbevers**
Also updates formatting in `GRIDS.md` for the newly added grids from Australia.

@sebastic can you give this a quick look before I merge? Just to make sure I don't add something rubbish...

### CODE REVIEW

File: GRIDS.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 ### Australia: AGD66 -> GDA94
 
-*Source*: ICSM - https://github.com/icsm-au/transformation_grids
+*Source*: [ICSM](https://github.com/icsm-au/transformation_grids)
 *Format*: NTv2
-*License*: Creative Commons Attribution 4.0
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
```
Comment (sebastic) [inline GRIDS.md]: Please don't mix these link changes for Australia in this PR, the link changes are unrelated to the new grids.

File: GRIDS.md [hunk 2]
```diff
@@ -104,3 +104,39 @@ Grid transformation from DE_DHDN to ETRS89 in Germany.
...
+
+Transform ellipsoidal heights to the local height system FVR09. The horizontal
+grid coordinates are referenced to ETRS89.
+
+* fvr09.gtx
+
+### Greenland: GRS80 ellipsoidal heights -> GVR2016
+
+
+
```
Comment (sebastic) [inline GRIDS.md]: Strip additional empty lines.

**Review by sebastic — CHANGES_REQUESTED** [review]
Please split the link changes for Australia into a separate commit (and preferably separate PR).

There are also some minor formatting consistency issues in `GRIDS.md`

File: GRIDS.md (hunk 1 above)
Comment (kbevers) [reply GRIDS.md]: I was being lazy. This is of course the correct way to proceed. I'll remove this for now.

**Comment by kbevers** [comment]
Thanks. I fixed your comments. 

> There are also some minor formatting consistency issues in GRIDS.md

I will take care of that later, including adding links in the Australia section.

**Review by sebastic — APPROVED** [review]
